src/main/java/microsoft/exchange/webservices/data/ServiceRequestBase.java

```diff
@@ -624,7 +624,11 @@ protected HttpWebRequest validateAndEmitRequest() throws ServiceLocalException,
     try {
       return this.getEwsHttpWebResponse(request);
     } catch (HttpErrorException e) {
-      processWebException(e, request);
+      try {
+        processWebException(e, request);
+      } finally {
```

Comment:
should the request be closed after return? meaning moving the finally to the outer catch-block?